ops/gauntlet/gauntlet_cosmos.go

```diff
@@ -0,0 +1,212 @@
+package gauntlet
```

Comment:
This file isn't used yet and I just brought it over from starknet. It will need to be updated to work with the cosmos gauntlet calls.